### PR TITLE
Release highlights for the week ending December 17

### PR DESCRIPTION
**Pull request by JosephSalisbury**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2021-12-17.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- [cert-manager-app](https://github.com/giantswarm/cert-manager-app/) [v2.12.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#2120---2021-12-16) upgrades the upstream image to [`v1.6.1`](https://github.com/jetstack/cert-manager/releases/tag/v1.6.1). This version stops serving cert-manager API versions `v1alpha2, v1alpha3, and v1beta1`. If you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process.
+- [external-dns-app](https://github.com/giantswarm/external-dns-app/) [v2.7.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#270---2021-12-16) upgrades to upstream [v0.10.2](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.10.2). This version removes support for Kubernetes versions <= 1.18.
+- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.5](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#235---2021-12-14) improves labels for monitoring.
```
Comment (QuentinBisson) [inline src/content/changes/highlights/2021-12-17.md:15]: I think you can add the whole EFK / Grafana and Prometheus Operator things here :)

Comment (marians) [inline src/content/changes/highlights/2021-12-17.md:15]:
```suggestion
- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.5](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#235---2021-12-14) improves labels for monitoring.

```

Comment (JosephSalisbury) [reply src/content/changes/highlights/2021-12-17.md]: thank you for your charity <3

**Review by puja108 — APPROVED** [review]

File: src/content/changes/highlights/2021-12-17.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- [prometheus-operator-app](https://github.com/giantswarm/prometheus-operator-app/) [v0.12.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#0120---2021-12-14) upgrades to upstream [23.2.0](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-23.2.0). This version fixes [CVE-2021-43798](https://nvd.nist.gov/vuln/detail/CVE-2021-43798).
+
+## User interfaces
+
+The new `update cluster` command in `kubectl-gs` allows to upgrade a workload cluster, either immediately or at some scheduled point in the future. See [the documentation](https://docs.giantswarm.io/ui-api/kubectl-gs/update-cluster/) for more details.
```
Comment (marians) [inline src/content/changes/highlights/2021-12-17.md:23]: No need for a code literal when mentioning the name of the software.

```suggestion
The new `update cluster` command in kubectl-gs allows to upgrade a workload cluster, either immediately or at some scheduled point in the future. See [the documentation](https://docs.giantswarm.io/ui-api/kubectl-gs/update-cluster/) for more details.
```

**Comment by marians** [comment]
In general, I'm not sure whether improved labels for monitoring quality for a "highlights" announcement. If you can make it a bit shorter by leaving those out, I'd vote for that.

File: src/content/changes/highlights/2021-12-17.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- [cert-manager-app](https://github.com/giantswarm/cert-manager-app/) [v2.12.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#2120---2021-12-16) upgrades to upstream [`v1.6.1`](https://github.com/jetstack/cert-manager/releases/tag/v1.6.1). This version stops serving cert-manager API versions `v1alpha2, v1alpha3, and v1beta1` - if you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process.
+- [efk-stack-app](https://github.com/giantswarm/efk-stack-app) [v0.7.1](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#071---2021-12-14) upgrades to upstream 1.13.3. This version fixes [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
+- [efk-stack-app](https://github.com/giantswarm/efk-stack-app) [v0.7.2](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#072---2021-12-16) correctly selects api version for ingress resources between `networking.k8s.io/v1` or `networking.k8s.io/v1beta1` depending on cluster version.
```
[the remark this answers is not in the document]
Comment (JosephSalisbury) [reply src/content/changes/highlights/2021-12-17.md:15]: It's a little repetitive, but I think it's worth it to explain the difference between the two. Especially with one being a CVE fix.

**Comment by JosephSalisbury** [comment]
sure, no strong opinion. dropped labelling improvement lines

**Pull request merged**